tests/test_project.py

```diff
@@ -250,6 +250,30 @@ def test_status_performance(self):
         assert time < 10
 
 
+class TestProjectStatusNoEligibleOperations(TestProjectBase):
+    class Project(FlowProject):
+        pass
+
+    @Project.operation
+    @Project.post(lambda job: True)
+    def foo(job):
+        pass
+
+    project_class = Project
+
+    def mock_project(self):
+        project = self.project_class.get_project(root=self._tmp_dir.name)
+        project.open_job(dict(i=0)).init()
```

Comment:
Shouldn't pyupgrade fix this?
```suggestion
        project.open_job({i: 0}).init()
```

Reply:
I think `dict(**kwargs)` is left alone. I believe pyupgrade changes the empty call `dict()` to a literal `{}` and it will rewrite dictionary comprehensions: https://github.com/asottile/pyupgrade#dictionary-comprehensions

Reply:
Ah I see.